### ISSUE TITLE
Doesn't work on windows

### ISSUE THREAD

**Issue by akriventsev** [AdRoll/baker#115]
I check the quick start example and get this error

```
# github.com/AdRoll/baker/input/inpututils
..\..\..\go\pkg\mod\github.com\!ad!roll\baker@v0.0.0-20210107173855-c0d2691ccc92\input\inpututils\fastreader.go:59:3: unknown field 'Setpgid' in struct literal of type syscall.SysProcAttr
```

**Comment by arl** [comment]
Thanks for reporting @akriventsev  will look into it.

**Comment by arl** [comment]
Closing, the issue has been fixed by #116
